compiler/testData/codegen/bytecodeText/stringOperations/constConcat.kt

```diff
@@ -0,0 +1,5 @@
+val s = "1" + "2" + 3 + 4L + 5.0 + 6F + '7'
```

Comment:
@pyos Please add similar test with const vals instead of const literals

val c = "${strConst}...${intConst}....